packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -453,6 +455,7 @@ Future<T> showModalBottomSheet<T>({
   Clip clipBehavior,
   bool isScrollControlled = false,
   bool useRootNavigator = false,
+  bool isDismissible = true,
```

Comment:
Update comment above to enumerate this option as well.

Reply:
Done.